app/src/main/java/com/github/se/cyrcle/ui/profile/ViewProfileScreen.kt

```diff
@@ -455,3 +463,126 @@ private fun UserReviewsSection(
+  // destinations paths of the images
+  val imagesPaths = userViewModel.selectedUserAssociatedImages.collectAsState().value
+  // Variables for Image/Parking in Alert Dialog
+  val showDialogImage = remember { mutableStateOf<String?>(null) }
+  val showDialogParking = remember { mutableStateOf<String?>(null) }
+  val showDialogImageDestinationPath = remember { mutableStateOf<List<String>>(emptyList()) }
+  val showDialogImagePath = remember { mutableStateOf<String?>(null) }
+
+  val strResToast = stringResource(R.string.view_profile_screen_image_deleted)
+
+  Box(modifier = Modifier.fillMaxSize()) {
+    if (imagesUrls.isEmpty()) {
+      Text(
+          text = stringResource(R.string.view_profile_screen_no_images_message),
+          style = MaterialTheme.typography.bodyMedium,
+          modifier = Modifier.padding(16.dp).testTag("NoImagesMessage"))
+    } else {
+      LazyColumn(
+          modifier = Modifier.fillMaxWidth().testTag("UserImagesList"),
+          contentPadding = PaddingValues(16.dp)) {
+            itemsIndexed(imagesUrls) { index, url ->
+              UserImageCard(
+                  url = url,
+                  onClick = {
+                    showDialogImage.value = url
+                    showDialogImageDestinationPath.value =
+                        showDialogImageDestinationPath.value.plus(it)
+                    showDialogParking.value =
+                        parkingViewModel.getParkingFromImagePath(imagesPaths[index])
+                    showDialogImagePath.value = imagesPaths[index]
+                  })
+            }
+          }
+    }
+  }
+
+  // Show dialog if an image is clicked
+  showDialogImage.value?.let { imageUrl ->
+    AlertDialog(
+        onDismissRequest = { showDialogImage.value = null },
+        title = { Text(stringResource(R.string.view_profile_screen_image_dialog_title)) },
+        text = {
+          Column {
+            Text(stringResource(R.string.view_profile_screen_image_dialog_description))
+            Spacer(modifier = Modifier.height(16.dp))
+            Image(
+                painter = rememberImagePainter(data = imageUrl),
+                contentDescription = null,
+                modifier = Modifier.fillMaxWidth().height(200.dp))
+          }
+        },
+        confirmButton = {
+          IconButton(
+              modifier = Modifier.padding(10.dp),
+              icon = Icons.Filled.LocalParking,
+              contentDescription = "Go To Parking",
+              testTag = "ParkingFromImageButton",
+              onClick = {
+                parkingViewModel.getParkingById(
+                    showDialogParking.value!!,
+                    { parkingForPath ->
+                      parkingViewModel.selectParking(parkingForPath)
+                      navigationActions.navigateTo(Screen.PARKING_DETAILS)
+                      Toast.makeText(context, strResToast, Toast.LENGTH_SHORT).show()
```

Comment:
I think this toast isn't supposed to be on this button, but rather on the other 😉

Reply:
Mixed things up!